x/fungible/keeper/evm.go

```diff
@@ -352,6 +353,7 @@ func (k Keeper) CallEVMWithData(
 	contract *common.Address,
 	data []byte,
 	commit bool,
+	noEthereumTxEvent bool,
```

Comment:
Would it be better to call it EthereumtxEvent? instead of checking !noEthereumTxEvent. I'm ok either way, just a bit confusing when reading.

Reply:
It's because not emitting ethereum tx event is the edge case, therefore I prefer having the zero value, false, to be used for the most usual case.

Maybe we can rename to `skipEthereumTxEvent` to have an affirmative name